packages/typescriptlang-org/src/components/layout/Sidebar.tsx

```diff
@@ -102,7 +102,7 @@ export const Sidebar = (props: Props) => {
         classes.push("closed")
       }
 
-      const opened = { "aria-expanded": "true", "aria-label": item.title + " close" }
```

Comment:
The naming implies this button can "close" things (because it's opened), the closed variable below has "expand" because it can expand things. Where does the bug come into play?

Reply:
It mentions that the Screen reader users will get incorrect information about the "Handbook button" as "Handbook Close Button

Reply:
Yes, so I checked in the office hours, and they said that "aria-label" is not needed here since the button already has a name. Just making sure that "aria-expanded" is correctly set to true or false will be enough for the screen reader to produce the expected result.